src/styles/_custom_variables.scss

```diff
@@ -21,6 +21,7 @@ $sidebar-z-index: 20;
 
 $header-logo-height: 80px;
 $header-logo-height-xs: 50px;
+$header-icon-color: $link-color;
```

Comment:
This doesn't work for me unless I change this to be:
`$header-icon-color: $link-color !default;`

In fact, none of the variables in this `_custom_variables.scss` seem to be possible to override in a custom theme, unless you first add a `!default` after them.